apps/web/src/containers/LandingPageView/components/our-products/hooks/use-lend-borrow-rates.tsx

```diff
@@ -0,0 +1,43 @@
+import { Market } from '@notional-finance/sdk/src/system';
+import { useAllMarkets } from '@notional-finance/notionable-hooks';
+
+export const useLendBorrowRates = () => {
+  const { cardData } = useAllMarkets();
+
+  let currentMaxRate = 0;
+  let currentMinRate: number | undefined = undefined;
+  let maxFixedLendRateData = {
```

Comment:
Would we rather return "undefined" here or some equivalent such that the rate does not show up at all? Not sure what will happen here during some loading state when there are no rates loaded yet.